interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -139,21 +138,32 @@
   private static final LinkedBlockingQueue<ServerStreamTracerInfo> serverStreamTracers =
       new LinkedBlockingQueue<ServerStreamTracerInfo>();
 
-  private static class ServerStreamTracerInfo {
+  private static final class ServerStreamTracerInfo {
     final String fullMethodName;
-    final ServerStreamTracer tracer;
+    final InteropServerStreamTracer tracer;
 
-    ServerStreamTracerInfo(String fullMethodName, ServerStreamTracer tracer) {
+    ServerStreamTracerInfo(String fullMethodName, InteropServerStreamTracer tracer) {
       this.fullMethodName = fullMethodName;
       this.tracer = tracer;
     }
+
+    private static final class InteropServerStreamTracer extends TestServerStreamTracer {
```

Comment:
yeah, i didn't have a good idea for this name.  If you want to send a PR i'd approve it.